docs/extensions/guides/catalog.md

```diff
@@ -1,5 +1,85 @@
-# Catalog (WIP)
+# Catalog
+Catalog is a special view in Lens IDE interface that aggregates information about important objects and entities.
```

Comment:
I had this as an introduction, maybe you can work some of it into yours:

The catalog was introduced in Lens 5.0 to provide cloud-native support beyond Kubernetes clusters.
Items that are contained in Lens's catalog are called entities.
Lens's built-in entities fall into two categories:
- Kubernetes clusters (`Common.Catalog.KubernetesCluster`), and 
- web links (`Common.Catalog.WebLink`).

The extension API provides extensive capabilities to 
- access entities in the catalog,
- add, inspect, and remove entities,
- add category-specific menu items for custom actions,
- define custom categories